WalletWasabi/Services/CcjClient.cs

```diff
@@ -753,8 +755,8 @@ private async Task DequeueCoinsFromMixNoLockAsync(params (uint256 txid, uint ind
 
 		private void RemoveCoin(SmartCoin coinWaitingForMix)
 		{
-			State.RemoveCoinFromWaitingList(coinWaitingForMix);
 			coinWaitingForMix.CoinJoinInProgress = false;
+			State.RemoveCoinFromWaitingList(coinWaitingForMix);
```

Comment:
Why modify the order here? I think it's not good, nor makes anything faster, if for some reason RemoveCoinFromWaitingList exceptions out CoinJoinInProgress should not be make false. I'll modify it, scream if you think otherwise.

Reply:
That change was pushed by mistake. I was working on another branch making sure that the coins listed in the "Coins Being CoinJoined" section were removed from that list and restored in the "Queue Coins for coinJoin" list when the server unqueue them.

That change is part of WIP and I pushed by mistake.